src/OneExplorer/ConfigObject.ts

```diff
@@ -19,6 +19,178 @@ import * as ini from 'ini';
+   * @param iniObj a parsed ini object
+   * @param dir a directory of ini file
+   * @returns an array of file path, without any duplication in the list
+   *
+   * EXAMPLE - file
+   *
+   * [one-import-tflite]            <--- section: "one-import-tflite"
+   *   input_file = "model.tflite"  <--- key: input_file
+   *   output_file = "model.circle" <--- key: output_file
+   *
+   * EXAMPLE - imported object
+   *
+   * iniObj[one-import-tflite]['input_file'] === "model.tflite"
+   * iniObj[one-import-tflite]['input_file'] === "model.circle"
+   */
+  public locate(iniObj: object, dir: string): string[] {
+    // Get file names from iniObj
+    const getFileNames = (): string[] => {
+      let fileNames: string[] = [];
+
+      const sections = this.section ? [this.section] : Object.keys(iniObj);
+      const sectionObjs = sections.map(section => iniObj[section as keyof typeof iniObj]);
+
+      sectionObjs.forEach(sectionObj => {
+        const keys: string[] = this.key ? [this.key] : Object.keys(sectionObj);
+
+        keys.forEach(key => {
+          const value: string = sectionObj[key as keyof typeof iniObj];
+          fileNames = fileNames.concat(this.mapper(value));
+        });
+      });
+
+      return fileNames;
+    };
+
+    // Get file paths by joining dir
+    const getFilePaths = (fileNames: string[]): string[] => {
+      let filePaths: string[] = [];
+      fileNames.forEach((fileName) => {
+        // In path, '..' or '//' are resolved here
+        const filePath = path.resolve(dir, fileName);
+
+        filePaths.push(filePath);
+      });
+
+      return filePaths;
+    };
+
+    // Get file names
+    let fileNames: string[] = getFileNames();
+
+    // Get file paths by joining directory path
+    let filePaths: string[] = getFilePaths(fileNames);
+
+    // Remove duplication in filePaths
+    filePaths = [...new Set(filePaths)];
+
+    return filePaths;
+  }
+};
+
+
+// TODO Move to backend side with some modification
+export class LocatorRunner {
+  private artifactLocators: {artifact: Artifact, locator: Locator}[] = [];
+
+  /**
+   * A helper function to grep a filename ends with 'ext' within the given 'content' string.
+   */
+  public static searchWithExt = (ext: string, content: string): string[] => {
+    // Don't remove this. It's to prevent an 'content.cplit is not a function' error.
+    // TODO Find more straightforward way to resolve an error
+    content = content + '';
+
+    const fileNames = content.split(' ').filter(val => val.endsWith(ext));
+    return fileNames;
+  };
+
+  constructor() {
+    this.artifactLocators.push({
+      artifact: {type: 'circle', ext: '.circle'},
+      locator: new Locator((value: string) => LocatorRunner.searchWithExt('.circle', value))
+    });
+
+    this.artifactLocators.push({
+      artifact: {type: 'tvn', ext: '.tvn'},
+      locator: new Locator((value: string) => LocatorRunner.searchWithExt('.tvn', value))
+    });
+  }
```

Comment:
```suggestion
    this.artifactLocators.push({
      artifact: {type: 'log', ext: '.circle.log'},
      locator: new Locator(
          (value: string) => {return LocatorRunner.searchWithExt('.circle', value)
                                  .map(filepath => filepath.replace('.circle', '.circle.log'))})
    });
  }
```
FOR EXAMPLE,
Above addition will support .circle.log